### PR TITLE
add presubmit for operator controller test

### PR DESCRIPTION
**Pull request by richardwxn**
pending on https://github.com/istio/istio/pull/22641, specified as optional now

### CODE REVIEW

File: prow/config/jobs/istio.yaml [hunk 1]
```diff
@@ -108,6 +108,19 @@ jobs:
       - name: TEST_SELECT
         value: "-postsubmit,-flaky,-multicluster"
 
+  - name: integ-operator-controller-tests
+    type: presubmit
+    command: [entrypoint, prow/integ-suite-kind.sh, test.integration.operator.kube.presubmit]
+    requirements: [kind]
+    modifiers: [optional]
```
Comment (howardjohn) [inline prow/config/jobs/istio.yaml]: please make it `[optional,hidden]` until the test has stabilized

File: prow/config/jobs/istio.yaml [hunk 2]
```diff
@@ -108,6 +108,19 @@ jobs:
...
+    env:
+      - name: HUB
+        value: "gcr.io/istio-testing"
+      - name: TAG
+        value: "1.6-dev"
```
Comment (howardjohn) [inline prow/config/jobs/istio.yaml]: don't set this, otherwise we are not testing the PR and instead are testing the already merged code

**Review by howardjohn — CHANGES_REQUESTED** [review]

File: prow/config/jobs/istio.yaml (hunk 2 above)
Comment (richardwxn) [reply prow/config/jobs/istio.yaml]: oh yes, but how can I specify hub and tag for this case? should I just leave it empty the test fw would infer?

Comment (howardjohn) [reply prow/config/jobs/istio.yaml]: the test will set hub and tag to `istio-testing` (not a real hub) and build images locally. Then the jobs will use those local images

Comment (richardwxn) [reply prow/config/jobs/istio.yaml]: got it

File: prow/config/jobs/istio.yaml (hunk 1 above)
Comment (richardwxn) [reply prow/config/jobs/istio.yaml]: done

**Review by dgn — APPROVED** [review]

**Review by howardjohn — APPROVED** [review]

**Pull request merged**